dmoj/executors/RUST.py

```diff
@@ -56,15 +57,40 @@ def create_files(self, problem_id, source_code, *args, **kwargs):
         with open(self._file('Cargo.toml'), 'wb') as f:
             f.write(CARGO_TOML)
 
+    shared_target = None
+
+    @classmethod
+    def get_shared_target(cls):
+        if cls.shared_target is None:
```

Comment:
If we invert this condition we can reduce nesting by one level.

Reply:
Updated.